template/_package.json

```diff
@@ -53,7 +53,9 @@
     "eslint-plugin-promise": "^3.5.0",
     "eslint-plugin-standard": "^3.0.1"<% } %><% if (ui === 'tailwind') { %>,
     "autoprefixer": "^7.1.6",
-    "tailwindcss": "^0.1.3"<% } %><% if (server !== 'none') { %>,
+    "tailwindcss": "^0.1.3"<% } else if (ui === 'vuetify') { %>,
+    "stylus": "^0.54.5",
+    "stylus-loader": "^3.0.1"<% } %><% if (server !== 'none') { %>,
```

Comment:
why stylus?

Reply:
oh I saw that..

Reply:
Because `vuetifyjs` uses `stylus` to build dynamic styles 😸